### DISCUSSION TITLE
Get Craft CMS to communicate with N8N

### DISCUSSION THREAD

**Discussion by Oceanestars** [craftcms/cms#12552] [Q&A]
Hi!
I'm very new to using both tools so I'm not 100% sure they can even communicate. I have an N8N workflow working but I just need to get the current logged in user which I get with Craft CMS but I can't figure out or find how I can get the two to communicate. Any help is greatly appreciated!!

**Comment by brandonkelly** [comment]
What exactly are you trying to accomplish?
